### PR TITLE
update: navigation and footer

### PR DESCRIPTION
**Pull request by sy-records**
#107

## Navigation

- [x] Add ASF dropdown menu, open in new window
- [x] Add Twitter menu, open in new window

## Footer

- [x] Add Disclaimer, Trademarks and more
- [x] Remove social media link

## Others

- [x] Stop link to externally hosted resources
> Didn't remove img.shields.io, which was used to get the number of stars.

### CODE REVIEW

File: src/theme/Footer/index.tsx [hunk 1]
```diff
@@ -1,78 +1,37 @@
...
-                  href="https://github.com/apache/incubator-answer/graphs/contributors"
-                  target="_blank"
-                >
-                  our contributors
-                </a>
-                .
+                <span>Apache Answer is an effort undergoing incubation at The Apache Software Foundation (ASF), sponsored by the Apache Incubator. Incubation is required of all newly accepted projects until a further review indicates that the infrastructure, communications, and decision making process have stabilized in a manner consistent with other successful ASF projects. While incubation status is not necessarily a reflection of the completeness or stability of the code, it does indicate that the project has yet to be fully endorsed by the ASF.</span>
+              </div>
+
+              <div style={{marginTop: '10px', marginBottom: '10px'}}>
```
Comment (fenbox) [inline src/theme/Footer/index.tsx:20]: use bootstrap default space class such as `my-3` here

File: src/theme/Footer/index.tsx [hunk 2]
```diff
@@ -1,78 +1,37 @@
...
   return (
     <footer className="footer">
       <div className="container py-3">
-        <div className="d-flex justify-content-between flex-wrap">
+        <div className="d-flex justify-content-between flex-wrap text-center">
           <div>
-            <div className='mb-3' style={{ height: '28px' }}>
-              <DarkSvg role="img" />
+            <div className='mb-3'>
+              <ApacheSvg role="img" />
```
Comment (fenbox) [inline src/theme/Footer/index.tsx:12]: logo need link to https://incubator.apache.org/

**Review by shuashuai — CHANGES_REQUESTED** [review]
According to the discussion, we currently do not need to support katex, so you can delete katex resources. After completion, we will merge this pr. Finally thank you for your contribution

File: docusaurus.config.js [hunk 3]
```diff
@@ -4,6 +4,16 @@ const lightCodeTheme = require('prism-react-renderer/themes/github');
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const math = require('remark-math');
 const katex = require('rehype-katex');
```
Comment (sy-records) [inline docusaurus.config.js:6]: Does `rehype-katex` need to be removed?

File: docusaurus.config.js
Comment (shuashuai) [inline docusaurus.config.js:1]: It doesn't seem to remove the reference to katex completely, you can remove the rehype-katex dependency by the way in package.json and then remove the reference to katex in docusaurus.config.js

image
image

File: docusaurus.config.js (hunk 3 above)
Comment (shuashuai) [reply docusaurus.config.js:6]: Of course, you can remove the dependencies of rehype-katex by the way.

Comment (sy-records) [reply docusaurus.config.js]: done.

**Pull request merged**